src/PropertyApp/MauiProgram.cs

```diff
@@ -1,16 +1,28 @@
-﻿using Microsoft.Extensions.Logging;
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
 using PropertyApp.JsonReadWrite;
 using PropertyApp.Model;
 using PropertyApp.ModelHandling;
 using PropertyApp.View;
 using PropertyApp.ViewModel;
+using System.Reflection;
 
 namespace PropertyApp;
 public static class MauiProgram
 {
     public static MauiApp CreateMauiApp()
     {
         var builder = MauiApp.CreateBuilder();
+
+        var assembly = Assembly.GetExecutingAssembly();
+        using var stream = FileSystem.OpenAppPackageFileAsync("appsettings.json");
```

Comment:
Check if anything is actually being used from the appsettings.json and if it can be completely removed